docs/getting-started.md

```diff
@@ -39,6 +30,38 @@ podman run -i --rm quay.io/coreos/fcct:release --pretty --strict < your_config.f
 podman run --rm -v /path/to/your_config.fcc:/config.fcc:z quay.io/coreos/fcct:release --pretty --strict /config.fcc > transpiled_config.ign
 ```
 
+You may also setup the following alias in your shell configuration:
+
+```
+alias fcct='podman run --rm --tty --interactive \
+            --security-opt label=disable        \
+            --volume ${PWD}:/pwd --workdir /pwd \
+            quay.io/coreos/fcct:release'
+```
+
+#### Distribution packages
+
+`fcct` is available from the Fedora packages repositories:
+
+```
+$ sudo dnf install -y fcct
+```
+
+Although this should work, we recommend that you run fcct from the container (as described in the previous section) to make sure that you always use the latest version. If you use the packaged version, make sure that your system is fully updated to the latest Fedora stable release.
```

Comment:
I think this is probably overstated.  We push new fcct releases to all supported Fedora releases, and there aren't generally adverse consequences from using older versions.  The main consequence is that newer specs may not be supported.